app/services/imports/lettings_logs_import_service.rb

```diff
@@ -200,13 +200,13 @@ def create_log(xml_doc)
       attributes["first_time_property_let_as_social_housing"] = first_time_let(attributes["rsnvac"])
       attributes["declaration"] = declaration(xml_doc)
 
-      attributes["uprn"] = string_or_nil(xml_doc, "UPRN")
-      attributes["uprn_known"] = attributes["uprn"].present? ? 1 : 0
-      attributes["uprn_confirmed"] = attributes["uprn"].present? ? 1 : 0
       attributes["address_line1"] = string_or_nil(xml_doc, "AddressLine1")
       attributes["address_line2"] = string_or_nil(xml_doc, "AddressLine2")
       attributes["town_or_city"] = string_or_nil(xml_doc, "TownCity")
       attributes["county"] = string_or_nil(xml_doc, "County")
+      attributes["uprn"] = address_given?(attributes) ? nil : string_or_nil(xml_doc, "UPRN")
+      attributes["uprn_known"] = attributes["uprn"].present? ? 1 : 0
+      attributes["uprn_confirmed"] = attributes["uprn"].present? ? 1 : 0
```

Comment:
if there's no uprn isn't the standard behaviour for `uprn_confirmed` to be nil?

Reply:
Hmm, I only moved it. I assume if it worked before it would retain the same behaviour